### PR TITLE
Script to load SNOMED Value sets in order to generate the procedures orders

### PR DESCRIPTION
**Pull request by nyatindopatrick**
## Requirements

- [x] This PR has a proper title that briefly describes the work done
- [x] I have squashed / amended the comments to make it more redable
- [x] I have included link to all the JIRA ticket('s)
- [x] I wrote the code as a pair or atleast performed a self-review of my own code

## Summary
Script to load SNOMED Value sets in order to generate the procedures orders

## JIRA tickets
https://bahmni.atlassian.net/browse/BS-109

### CODE REVIEW

File: procedures/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+   * [how to install NodeJS on mac](https://www.newline.co/@Adele/how-to-install-nodejs-and-npm-on-macos--22782681)
+
+### Environment Variables
+
+| Variable           | Description                     | Example                                                                       |
+| ------------------ | ------------------------------- | ----------------------------------------------------------------------------- |
+| SAVE_VALUESET_URL  | Snowstorm server value set url | https://snowstorm.snomed.mybahmni.in/fhir/ValueSet                            |
+| BAHMNI_FHIR_SERVER | Bahmni procedures endpoint      | https://{bahmni.instance.com}/openmrs/ws/rest/v1/terminologyServices/valueSet |
+| USERNAME           | Bahmni username                 | superman                                                                      |
+| USERNAME           | Bahmi password                  | Admin123                                                                      |
```
Comment (gsluthra) [inline procedures/README.md:25]: typo. Should be password.

File: procedures/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+   * [how to install Node using nvm](https://github.com/nvm-sh/nvm).
+   * [how to install NodeJS on mac](https://www.newline.co/@Adele/how-to-install-nodejs-and-npm-on-macos--22782681)
+
+### Environment Variables
+
+| Variable           | Description                     | Example                                                                       |
+| ------------------ | ------------------------------- | ----------------------------------------------------------------------------- |
+| SAVE_VALUESET_URL  | Snowstorm server value set url | https://snowstorm.snomed.mybahmni.in/fhir/ValueSet                            |
+| BAHMNI_FHIR_SERVER | Bahmni procedures endpoint      | https://{bahmni.instance.com}/openmrs/ws/rest/v1/terminologyServices/valueSet |
```
Comment (gsluthra) [inline procedures/README.md:23]: Should be called just BAHMNI_SERVER_URL

File: procedures/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Procedures Utility
+
+This repository contains the utility code for adding value sets to the snowstorm server.
```
Comment (gsluthra) [inline procedures/README.md:3]: Doesn't this utility ALSO add data into Bahmni Server? If so, then the description needs to be updated.

File: procedures/README.md (hunk 2 above)
Comment (gsluthra) [inline procedures/README.md:23]: Is this a new REST endpoint we added in Bahmni via the snomed-omod?

Comment (nyatindopatrick) [reply procedures/README.md]: Updated

File: procedures/README.md (hunk 1 above)
Comment (nyatindopatrick) [reply procedures/README.md]: Fixed

File: procedures/README.md (hunk 3 above)
Comment (nyatindopatrick) [reply procedures/README.md]: Updated

File: procedures/README.md (hunk 2 above)
Comment (nyatindopatrick) [reply procedures/README.md]: Yes

File: procedures/README.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+1. Install node/npm (node version: 10.11.0). Preferably use nvm, so that you have control over which project uses which version of node. See:
+
+   * [how to install Node using nvm](https://github.com/nvm-sh/nvm).
+   * [how to install NodeJS on mac](https://www.newline.co/@Adele/how-to-install-nodejs-and-npm-on-macos--22782681)
+
+### Environment Variables
+
+| Variable          | Description                     | Example                                                                       |
+| ----------------- | ------------------------------- | ----------------------------------------------------------------------------- |
+| SAVE_VALUESET_URL | Snowstorm server value set url | https://snowstorm.snomed.mybahmni.in/fhir/ValueSet                            |
```
Comment (angshu) [inline procedures/README.md:22]: better name than SAVE_VALUESET_URL? why not just VALUESET_URL?

Comment (nyatindopatrick) [reply procedures/README.md]: This has been updated to VALUESET_URL

**Review by angshu — APPROVED** [review]

**Pull request merged**